File: fix_syntax.js
import fs from 'fs';
let code = fs.readFileSync('src/App.tsx', 'utf8');

code = code.replace(/          onPlayerInteract=\{\(playerId, playerName\) => \{\n            setInteractPlayerId\(playerId\);\n            setInteractPlayerName\(playerName\);\n          \}\} : null;\n              \}\n              return next;\n            \}\}\n          onBlockMined/g, 
`          onPlayerInteract={(playerId, playerName) => {
            setInteractPlayerId(playerId);
            setInteractPlayerName(playerName);
          }}
          onBlockMined`);

fs.writeFileSync('src/App.tsx', code);

File: fix_syntax.py
lines = open('src/App.tsx').read().split('\n')
for i in range(2200, 2350):
    if "''}\">" in lines[i]:
        lines[i] = lines[i].replace("''}\">", "''}`>")
    if "'z-20'}\">" in lines[i]:
        lines[i] = lines[i].replace("'z-20'}\">", "'z-20'}`>")

open('src/App.tsx', 'w').write('\n'.join(lines))
